### PR TITLE
build: bump ovh-api-services to v6.16.0

### PR DESCRIPTION
**Pull request by marie-j**
# Bump ovh-api-services to v6.16.0

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1688,6 +1688,17 @@
   dependencies:
     lodash "^4.17.11"
 
+"@ovh-ux/ng-ovh-cloud-universe-components@^1.2.0-alpha.0":
```
Comment (cbourgois) [inline yarn.lock]: Seems not related to the `ovh-api-services` bump

Comment (antleblanc) [reply yarn.lock]: Please refer to this Pull request first: #494

Comment (marie-j) [reply yarn.lock]: I'll rebase and update once https://github.com/ovh-ux/manager/pull/494 is merged

Comment (antleblanc) [reply yarn.lock]: It is now merged:)
